NewUser'sGuide.md

```diff
@@ -0,0 +1,250 @@
+# New User’s Guide to Lutris
+
+Welcome to Lutris, [a game library manager](https://github.com/lutris/lutris) that allows you to bring your entire game collection together regardless of platform.
+
+This is a guide for new users that are just starting out with Lutris. Once you have [installed the Lutris client](https://lutris.net/downloads) for your distribution, [updated your Nvidia and AMD/Intel drivers](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) and have a [recent version of WINE](https://wiki.winehq.org/Download), you’ll be ready to go on your Lutris journey.
+
+## Installing a Game
+
+In Lutris, there are many ways to install games from your game collections.
+
+### Manual Game Installation
```

Comment:
We discourage manual installation so this section shouldn't in a guide for new users.